pytorch_lightning/utilities/xla_device_utils.py

```diff
@@ -0,0 +1,57 @@
+import functools
+import importlib
+from multiprocessing import Process, Queue
+
+TORCHXLA_AVAILABLE = importlib.util.find_spec("torch_xla") is not None
+if TORCHXLA_AVAILABLE:
+    import torch_xla.core.xla_model as xm
+else:
+    xm = None
+
+
+def inner_f(queue, func, **kwargs):
```

Comment:
```suggestion
def inner_f(queue, func, **kwargs):  # pragma: no cover
```
I am not 100% sure, but we may want to ignore the codecoverage on some of these multiprocessing parts.
For example, inner_f is launched with a new process, I'm pretty sure codecoverage cannot reach into it.